src/Nexus/Services/DatabaseService.cs

```diff
@@ -72,12 +72,23 @@ internal class DatabaseService(IOptions<PathsOptions> pathsOptions)
 
     private readonly PathsOptions _pathsOptions = pathsOptions.Value;
 
+    private readonly string _users = "users";
```

Comment:
These kinds of fields should be `const string` instead of `readonly string` and then they should be named with uppercase letters only like `USERS` or `USERS_FOLDER`

Reply:
Done :-)